tests/commonTemplates_test.go

```diff
@@ -656,6 +656,18 @@ var _ = Describe("Common templates", func() {
 							},
 						}),
 				)
+				It("[test_id:4878]should not create any other resurces than the ones listed in the Edit Cluster role", func() {
```

Comment:
We have duplicate entries of the test_id: 4876 - https://github.com/kubevirt/ssp-operator/blob/master/tests/commonTemplates_test.go#L638-L657. I thought the 4878 block was going to replace one of those duplicate blocks. But looks like it is not. Could you remove the duplicate entry?

Reply:
Removed it now